dotnet/src/SemanticKernel/CoreSkills/SemanticFunctionConstants.cs

```diff
@@ -440,4 +482,74 @@ foo bar
 [Input]
 {{$INPUT}}
 [Output]";
+    internal const string ParseConditionGroup =
+        @"<conditiongroup>
+    <condition variable=""x"" exact=""1"" />
+    <and/>
+    <condition variable=""y"" contains=""asd"" />
+    <or/>
+    <not>
+    <conditiongroup>
+        <condition variable=""z"" greaterthan=""10"" />
+    </conditiongroup>
+    </not>
```

Comment:
Yeah, way too complex. These compound conditions would be difficult to maintain. Please cut the conditiongroup for this first version -- keep it simple.